exchange/exchangetest/bid-id-invalid.json

```diff
@@ -88,6 +88,7 @@
                 "IAB1-1"
               ],
               "ext": {
+                "origbidcpm": 0,
```

Comment:
It looks like this is set to `0` instead of `0.3` because the bidder adapters used by the exchange created in the tests are of type `validatingBidder`, which is test-specific, rather than `adaptedBidder` which is used in production. `validatingBidder` has its own implementation of `requestBid`. The `origbidcpm` is set to the bid response price in the `adaptedBidder` `requestBid` implementation which is not used during the test.
It's a little strange that this doesn't match the price in the mock response. I'm not sure if there's an easy fix here. Perhaps the `validatingBidder` can be modified?

Reply:
Yeah I also noticed this while updating these tests, but because of this difference in what the test is using vs. what production is using, you get this behavior. 

I have already written tests that properly test the implementation of this PR, so we already covered in that sense.

But I agree that it's strange to see this, I could look into if validatingBidder can be modified, but will bring up in the meeting if this is worth it

Reply:
After looking to the code, it looks like it's a real easy change to modify validatingBidder to properly handle the origbidcpm implementation. I'll just update it!